_source/_posts/2023-06-08-identity-developer-podcast.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Unlock identity expertise with our new developer podcast!"
+author: edunham
+by: advocate
+communities: [devops]
```

Comment:
I'd recommend adding other communities unless this is truly only devops based podcast